templates/cf-apache.service.mustache

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=CFEngine Enterprise Webserver
+After=syslog.target
+After=cf-postgres.service
+ConditionPathExists={{{vars.sys.workdir}}}/httpd/bin/apachectl
+PartOf=cfengine3.service
+
+[Service]
+Type=forking
+ExecStart={{{vars.sys.workdir}}}/httpd/bin/apachectl start
+ExecStop={{{vars.sys.workdir}}}/httpd/bin/apachectl stop
+PIDFile={{{vars.sys.workdir}}}/httpd/logs/httpd.pid
+Restart=always
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
+WantedBy=cfengine3.service
```

Comment:
Yes, I have the same thing in my own notes. This is actually disabled right now (the bundles are never called). I was still debating on actively managing the unit states by default or not. Figured I would start off with not having them active and while testing was going on.